.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,58 @@
+name: Makefile and Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "*"
+
+jobs:
+  unit_test_without_bcc:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@main
+      with:
+          go-version: 1.18
+    - name: Get ginkgo
+      run: make ginkgo-set
+      env:
+          GOPATH: /home/runner/go
+          GOBIN: /home/runner/go/bin
+    - name: Run 
+      run: make test-verbose
+
+  unit_test_with_bcc:
+    runs-on: ubuntu-latest
```

Comment:
for future work, would it be possible to use a custom container image (i.e. the kepler base image) to run the test? It would save the bcc build stage.